### ISSUE TITLE
Luk 20:42 - the book/The Book of Psalms

### ISSUE THREAD

**Issue by lrsallee** [Door43/ulb-en#106]
ULB does not capitalize " the book" but UDB does.

**Comment by TomWarren** [comment]
This was fixed (by whom?). Ran a search, and there are no more "Book of Psalms" references in the UDB.

Is it time to make the names of the Books of the USFM follow consistent naming conventions? What a mess that is.

**Comment by lrsallee** [comment]
My bad. I reversed the versions. ULB is capital and UDB is not. Problem remains.

**Comment by TomWarren** [comment]
The convention is that we follow the ULB. I will see about bringing them into alignment. There is also the problem of "The Book of Psalms" and "the Book of Psalms".
Will take a minute to fix.

**Comment by lrsallee** [comment]
Hard to follow the ULB when it is still inconsistent.

**Comment by lrsallee** [comment]
What confuses me is the capitalization of the other books as well. If you drop off "Psalms" and do a ULB NT search for "the book of" you will see quite a range.

**Comment by TomWarren** [comment]
We need to devote just a few minutes to the naming conventions across the board, from quotations (like here) to the very names of the books in the USFM. 
It is a mess.

**Comment by lrsallee** [comment]
Agreed. Book of Life is another one that is erratic.

**Comment by TomWarren** [comment]
Just found 2x in Rev "The Book of Life" changed to "the Book of Life". Following what seems to be the convention that books do not capitalize the "The".
There is no "Book of Life" or "book of Life" in the UDB (interesting tid bit). But it did have "book of life" 5x. 
All made consistent with the ULB.

**Comment by TomWarren** [comment]
I searched for "the book" and covered all of them. Should be getting closer to a naming convention.
